CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [0.2.0] - 2020-01-09
+
 ## [0.2.1] - 2019-11-19
 
 ## [0.2.0] - 2019-11-18
```

Comment:
How are you managing releases? Why there is two `0.2.0`? Since the project wasn't launched, maybe we should be evolving only the patch number (e.g.: `0.0.1`, `0.0.2`,...)

Reply:
I never manually touched this file, I always used the checkboxes from [vtex-io-ci-cd](https://github.com/apps/vtex-io-ci-cd) . I agree with only evolving the patch number, but how to fix that now? Is it just changing this file?

Reply:
You have to add to `CHANGELOG.md` any changes you make on every PR. 

Check [this link](https://keepachangelog.com/en/1.0.0/) to see the format.

Reply:
Also you can see on another repo how we usually increment the `CHANGELOG.md`.

Reply:
Oh, silly me, I'll read that now, I thought everything was automatically generated (e.g. from commit names). Thanks!

Reply:
Oh, literally the first line of that link is "Don’t let your friends dump git logs into changelogs" 😆.